libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -234,6 +234,20 @@ - (void)testGetDescribeWithObjectType {
     XCTAssertEqualObjects(listener.returnStatus, kTestRequestStatusDidLoad, @"request failed");
 }
 
+// simple: just invoke requestForLayoutWithObjectType:@"Contact" without layoutType.
+- (void)testGetLayoutWithObjectTypeWithoutLayoutType {
```

Comment:
Tests for new REST API wrapper - 1 without `layoutType` (default case) and 1 with.